build/Dockerfile

```diff
@@ -61,8 +62,8 @@ ENV USER_NAME=psap-ci-runner \
 COPY . ${HOME}/
 
 # Install Python requirements
-RUN python3 -m pip install --quiet --no-cache-dir -r ${HOME}/requirements.txt && \
-    python3 -m pip install --quiet --no-cache-dir -r ${HOME}/subprojects/wdm/requirements.txt
+RUN venv/bin/pip/pip3 install --no-cache-dir -r ${HOME}/requirements.txt && \
```

Comment:
That's where it fails, because:
```
 venv/bin/pip/pip3 install --no-cache-dir -... --no-cache-dir -r ${HOME}/subprojects/wdm/requirements.txt
/bin/sh: venv/bin/pip/pip3: Not a directory
```
(the path is wrong)
I'll run locally tomorrow first thing